mods/cubyz/rotation/log.zig

```diff
@@ -242,9 +239,6 @@ pub fn updateData(block: *Block, neighbor: Neighbor, neighborBlock: Block) bool
 		currentData.setConnection(neighbor, neighborData.isConnected(neighbor.reverse()));
 	}
 
-	const result: u16 = currentData.enabledConnections;
-	if(result == block.data) return false;
```

Comment:
I think we should keep this optimization on updateData, right? This doesn't seem related to the bug.